File: python/LengthofLastWord.py
class LengthofLastWord(object):
    def lengthOfLastWord(self,s):
        '''
        :type s:str
        :rtype:int
        '''
        if s=="":
            return 0
        n=0
        i=len(s)-1
        while i>=0 and s[i]=='':
            i=i-1
        print i
        while i>=0 and s[i]!='':
            i=i-1
            n=n+1
        return n

L=LengthofLastWord()
print L.lengthOfLastWord("qwer  ")
            

File: C/LengthofLastWord.c
#include <string.h>

int lengthOfLastWord(char* s)
{
	int i = strlen(s), j = 0;
	for (--i; i >= 0 && s[i] == ' ';)
		--i;
	for (j = i; j >= 0 && s[j] != ' ';)
		--j;
	return i - j;
}
